rest_framework/fields.py

```diff
@@ -1105,7 +1106,10 @@ def enforce_timezone(self, value):
         field_timezone = getattr(self, 'timezone', self.default_timezone())
 
         if (field_timezone is not None) and not timezone.is_aware(value):
-            return timezone.make_aware(value, field_timezone)
+            try:
+                return timezone.make_aware(value, field_timezone)
+            except Exception:
```

Comment:
Here `pytz` raises `AmbiguousTimeError` or `NonExistentTimeError` both have parent `pytz.exceptions.InvalidTimeError(Exception)`.
django-rest-framework doesn't have 'pytz` in requirements (except -testing added in this PR) and I think it's not worth to depend on it here only to catch specific exceptions. You can not have `pytz` but still can use drf.